add project name to docker-compose

rjw57 opened the pull request:
docker-compose uses the project name as a prefix for all Docker resources it creates. This defaults to the name of the directory holding the compose file. This is not great in our use case because all our compose files live inside a directory named "compose". Specify the docker-compose project name explicitly in the ./compose.sh wrapper.

rjw57 committed:
add project name to docker-compose

docker-compose uses the project name as a prefix for all Docker
resources it creates. This defaults to the name of the directory holding
the compose file. This is not great in our use case because all our
compose files live inside a directory named "compose". Specify the
docker-compose project name explicitly in the ./compose.sh wrapper.

msb reviewed (dismissed):
Works fine - so can I suggest we rename the volume back to their generic name. The volumes are:
 - gauth-webapp-postgres-data-local
 - gauth-webapp-postgres-backup-local

rjw57 committed:
remove namespaced prefix for volume names

With docker-compose now correctly prefixing volume names, we don't have
to. Remove the project-specific prefix from volume names.
